flamingpy/_version.py

```diff
@@ -15,4 +15,4 @@
 """Version number (major.minor.patch[label])"""
 
 
-__version__ = "0.6.0a3"
+__version__ = "0.6.0a3.dev1"
```

Comment:
(minor) Just to conform with other PSS packages.
```suggestion
__version__ = "0.6.0a3-dev"
```
or
```suggestion
__version__ = "0.6.0-dev"
```

Reply:
Thanks for the suggestions @thisac. However, [PEP440](https://peps.python.org/pep-0440/) preferred way of doing this is using a `.` to separate the `dev` part rather than a `-` (if you use the latter, I believe it normalizes back to the pythonic version with a warning). We also want to have numbering after dev to change after each PR in-between the releases. Otherwise, there will be say two "0.6.0a3.dev" versions in the user's hand with completely different files, which is not ideal.

A reminder that our current version bump rules are very simple: we use semantic+PEP440 with every release and in-between releases CODEOWNERS update this with each PR as in adding `.dev1`, then `.dev2`, ...

@ilan-tz, what's your view here? Happy to meet with all to discuss this further.

Reply:
Thanks @nariman87. As per our discussion, I'm happy keeping the current system.